patch_script.cjs

```
@@ -1,62 +1,130 @@
-code = code.replace(
-    /case 'KeyQ':\n\s+if \(getIsPlaylistOpen\(\)\) return; \/\/ Handled by playlist manager\n\s+triggerButtonPressDown\('openJukeboxBtn'\);/,
-    `case 'KeyQ':\n                if (getIsPlaylistOpen()) return; // Handled by playlist manager\n                if (event.repeat) return;\n                triggerButtonPressDown('openJukeboxBtn');`
+if (!content.includes("isPhotoModeActive")) {
+    content = content.replace(
+        "import { getPhotoMode } from '../systems/photo-mode/index.ts';",
+        "import { getPhotoMode, isPhotoModeActive } from '../systems/photo-mode/index.ts';"
+    );
+}
+
+content = content.replace(
+    /updateFoliagePhase\([\s\S]*?dayNightBias\n    \);/,
```

ESLint

[error] Spaces are hard to count. Use {4}.

(no-regex-spaces)